nodejs18.x/full-stack/{{cookiecutter.project_name}}/backend/src/handlers/get-all-items.mjs

```diff
@@ -0,0 +1,61 @@
+// Create clients and set shared const values outside of the handler.
+
+// Create a DocumentClient that represents the query to add an item
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient, ScanCommand } from '@aws-sdk/lib-dynamodb';
+
+//DynamoDB Endpoint
+const ENDPOINT_OVERRIDE = process.env.ENDPOINT_OVERRIDE;
+if(!ENDPOINT_OVERRIDE){
+  console.warn("No value for ENDPOINT_OVERRIDE provided for DynamoDB");
+}
+
+const client = new DynamoDBClient({ endpoint: ENDPOINT_OVERRIDE });
```

Comment:
can we skip providing the endpoint if `ENDPOINT_OVERRIDE` is an empty string, to use the default value. (I am not sure if SDK client will ignore the empty string and use the default endpoint or not)

Reply:
change made as requested

Reply:
Thank you for the feedback, this is addressed and updated.